packages/node_modules/@webex/plugin-meetings/src/metrics/config.js

```diff
@@ -72,65 +72,66 @@ const errorFailureType = {
   // Media events MUST have the mediaType property.
-  CLIENT_MEDIA_RX_STOP: 'client.media.rx.stop',
+  RECEIVING_MEDIA_STOP: 'client.media.rx.stop',
   // Fired when the media engine reports sending a new media stream. Media events MUST have the mediaType property.
-  CLIENT_MEDIA_TX_START: 'client.media.tx.start',
+  SENDING_MEDIA_START: 'client.media.tx.start',
   // Fired when the media engine reports it stopped sending a media stream.
   // Media events MUST have the mediaType property.
-  CLIENT_MEDIA_TX_STOP: 'client.media.tx.stop',
-  CLIENT_MEDIA_RENDER_START: 'client.media.render.start',
-  CLIENT_MEDIA_RENDER_STOP: 'client.media.render.stop',
+  SENDING_MEDIA_STOP: 'client.media.tx.stop',
+  MEDIA_RENDER_START: 'client.media.render.start',
+  MEDIA_RENDER_STOP: 'client.media.render.stop',
   // static media event when outside of the normal scenario
   // call-analyzer assumes that a client is capable of receiving audio, video, and share
   // fired on change, or at beginning of a call
   // every media type is required, so must be indicated with boolean
-  CLIENT_MEDIA_CAPABILITIES: 'client.media.capabilities',
+  MEDIA_CAPABILITIES: 'client.media.capabilities',
   // Sent when the client notices that a media session has been lost
-  CLIENT_MEDIA_RECONNECTING: 'client.media.reconnecting',
+  MEDIA_RECONNECTING: 'client.media.reconnecting',
   // Sent when the client recovers a media session that was lost
-  CLIENT_MEDIA_RECOVERED: 'client.media.recovered',
+  MEDIA_RECOVERED: 'client.media.recovered',
+  MEDIA_RECONNECTION_FAILED: 'client.media.recovered',
   // Fired when the "please enter your PIN" or similar prompt is displayed
   // to the user, to authenticate them into the meeting
-  CLIENT_PIN_PROMPT: 'client.pin.prompt',
+  PIN_PROMPT: 'client.pin.prompt',
   // Fired when PIN entry has been completed
-  CLIENT_PIN_COLLECTED: 'client.pin.collected',
+  PIN_COLLECTED: 'client.pin.collected',
   // Fired when the client displays the native lobby
-  CLIENT_LOBBY_ENTERED: 'client.lobby.entered',
+  LOBBY_ENTERED: 'client.lobby.entered',
   // Fired when the client leaves the native lobby
-  CLIENT_LOBBY_EXITED: 'client.lobby.exited',
+  LOBBY_EXITED: 'client.lobby.exited',
   // Fired when the user of the client starts a share (e.g. click 'Share' button).
   // This should be sent from all clients that support sending a share.
-  CLIENT_SHARE_INITIATED: 'client.share.initiated',
+  SHARE_INITIATED: 'client.share.initiated',
   // Fired when the user stops sharing (usually when they click the 'Stop' button for share)
-  CLIENT_SHARE_STOPPED: 'client.share.stopped',
+  SHARE_STOPPED: 'client.share.stopped',
   // When the client receives a successful response from locus indicating that it has the floor for content sharing.
-  CLIENT_SHARE_FLOOR_GRANTED_LOCAL: 'client.share.floor-granted.local',
+  SHARE_FLOOR_GRANTED_LOCAL: 'client.share.floor-granted.local',
```

Comment:
I would place `LOCAL_` at the front to be consistent with the first constant above.